libs/community/langchain_community/utilities/jira.py

```diff
@@ -14,6 +15,7 @@ class JiraAPIWrapper(BaseModel):
     jira_username: Optional[str] = None
     jira_api_token: Optional[str] = None
     jira_instance_url: Optional[str] = None
+    jira_cloud: Optional[str] = "True"
```

Comment:
can we make this be a boolean? this is the instance on the model. if we load jira_cloud from dict_or_env we can convert from string to boolen there